src-java/server42/server42-control/src/main/java/org/openkilda/server42/control/kafka/Gate.java

```diff
@@ -98,6 +100,12 @@ public Gate(@Autowired KafkaTemplate<String, Object> template,
         ).collect(Collectors.toMap(SimpleEntry::getKey, SimpleEntry::getValue));
     }
 
+    @Override
+    public void onPartitionsAssigned(Map<TopicPartition, Long> assignments, ConsumerSeekCallback callback) {
+        log.info("seek to end");
```

Comment:
Can we add some info about topic / partition to the logged message?

Reply:
Info about consumer group id is added to the message.

example:
` 2021-06-30T16:26:39,368 INFO  [server42-control-0-C-1] o.o.s.c.k.Gate - [{}] seek to end`